src/main/java/Main.java

```diff
@@ -100,23 +103,53 @@ public static int CrudOption() {
 
     }
 
-    public static void CreatePatient(){
+    public static void createPatient(PatientClass patient,String DatabaseName){
```

Comment:
```suggestion
    public static void createPatient(int Index){
    String DatabaseName = tables[index]
    //Then insert the DatabaseName inside the CreatePatient with the Patient Class 
```

By the way in my class I already created the index of the database they wish to access with ch variable and you can use that variable to get the database from the array so the code will look something like the suggestion 
that is one way but if you have another way you can go and do it  however I think the easier and nicer way is to use the cb variable

Reply:
perfect thank you!